### PR TITLE
fix(mongo): upgraded mongo (LLC-473)

### PR DESCRIPTION
**Pull request by cbishopvelti**

### CODE REVIEW

File: src/service/getPersonasConnection.ts [hunk 1]
```diff
@@ -10,8 +10,13 @@ import Config from './Config';
 
 export default (config: Config) => {
   return async (opts: GetPersonasConnectionOptions): Promise<GetPersonasConnectionResult> => {
+    if (opts.limit === undefined) {
```
Comment (ryansmith94) [inline src/service/getPersonasConnection.ts:13]: I'm not sure I understand the need for this change. The previous code would have handled this already, was there a typing issue?

**Comment by ryansmith94** [comment]
Seems to be failing the linter at the moment.

File: src/service/getPersonasConnection.ts (hunk 1 above)
Comment (cbishopvelti) [reply src/service/getPersonasConnection.ts:13]: Yes, there is a test for if limit is undefined. According to the type, it should never be undefined. New tsc didn't like the spread as limit should always be defined according to its type.

Comment (ryansmith94) [reply src/service/getPersonasConnection.ts]: I see, @cbishopvelti should we just correct the type?

**Pull request closed without merging**